.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 *.log
+samples/**/Secrets.gs
```

Comment:
Can you mention where we create this `Secrets.gs` file in the README or in a comment here?